.env

```diff
@@ -0,0 +1,12 @@
+MYSQLDB_USER=root
```

Comment:
It's better to create .env.sample file with 


```
MYSQLDB_USER=/your db user/
MYSQLDB_ROOT_PASSWORD=/your db password/
MYSQLDB_DATABASE=/your db name/
MYSQLDB_LOCAL_PORT=/your db local port/
MYSQLDB_DOCKER_PORT=3306

SPRING_LOCAL_PORT=8088
SPRING_DOCKER_PORT=8080
DEBUG_PORT=5005

JWT_SECRET=/your jwt-secret/
JWT_EXPIRATION=/your jwt-expiration/
```

Reply:
Done